utils/pointer/pointer.go

```diff
@@ -0,0 +1,11 @@
+package pointer
```

Comment:
No need to create your own custom package for testing, when the original code already uses a pointer package.